meteor/server/databaseMigration.ts

```diff
@@ -76,11 +76,14 @@ export function addMigrationStep (step: MigrationStep) {
  * @param version
  * @param steps
  */
-export function addMigrationSteps (version: string, steps: Array<MigrationStepBase>) {
+export function addMigrationSteps (version: string, steps: Array<MigrationStepBase | null>) {
 	_.each(steps, (step) => {
-		addMigrationStep(_.extend(step, {
-			version: version
-		}))
+		if (step) {
+			addMigrationStep(_.extend(step, {
+				id: version + step.id,
```

Comment:
@nytamin as discussed

Reply:
I'm not convinced this should be added at the "framework"-level.
I'd rather have it at the "user"-level (ie where the steps are defined)